src/RemoteFileAPI/RemoteFileAPI.ts

```diff
@@ -5,8 +5,9 @@ let server: Remote;
 
 export function newRemoteFileApiConnection(): void {
   if (server) server.stopConnection();
-  if (Settings.RemoteFileApiPort === 0) return;
-  server = new Remote("localhost", Settings.RemoteFileApiPort);
+  if (Settings.RemoteFileApiAddress === "") Settings.RemoteFileApiAddress = "localhost";
```

Comment:
Don't change the value in settings, that's (potentially) permanent. Set a local instead.

Reply:
I was thinking about this line as more of return to default value in settings and connect.

Reply:
Switched logic to display "localhost" placeholder instead switching it in Settings. 
Connection with empty string to localhost is now done with Ternary.